### PR TITLE
Literate: init at 2017-05-28

### PR DESCRIPTION
**Pull request by RocketPuppy**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by RocketPuppy** [comment]
Odd that Travis reported a bad sha for the download `1dz5ifbkmwmah6s6s7ai8g9py9x3lk9fa1d5f4mwz6ng7qsfz7ci`, but that sha worked just fine locally. I tried the sha that Travis wanted as well, `094lramvacarzj8443ns18zyv7dxnivwi7kdk5xi5r2z4gx338iq` which also worked locally. I wonder if it is related to the git submodule?

File: pkgs/development/tools/literate-programming/Literate/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  buildInputs = [ dmd dub ];
+
+  installPhase = "install -D bin/lit $out/bin/lit";
+
+  meta = with stdenv.lib; {
+    description = "A literate programming tool for any language";
+    homepage    = http://literate.zbyedidia.webfactional.com/;
+    license = licenses.mit;
+    platforms = platforms.unix;
```
Comment (Mic92) [inline pkgs/development/tools/literate-programming/Literate/default.nix]: I adapted your package a little to fit our coding standards. Do you want to maintain it?

Comment (RocketPuppy) [reply pkgs/development/tools/literate-programming/Literate/default.nix]: Thank you. I appreciate the corrections. I notice the CI is still failing. I'm not sure of the best way to proceed regarding that. The specific failure I'm not familiar with. Do you have any suggestions?

Comment (Mic92) [reply pkgs/development/tools/literate-programming/Literate/default.nix]: ignore the CI, for the moment.

**Comment by joachifm** [comment]
@GrahamcOfBorg build Literate

**Pull request merged**